dcos/marathon.py

```diff
@@ -747,3 +757,19 @@ def _default_marathon_error(message=""):
     return ("Marathon likely misconfigured. Please check your proxy or "
             "Marathon URL settings. See dcos config --help. {}").format(
                 message)
+
+
+def _normalize_marathon_id_path(id_path):
```

Comment:
I actually found another implementation of this in `dcos.util`, so I moved this implementation there, and updated all usages of it.